### DISCUSSION TITLE
Request has been approved but benefits are not accessible after 5 days.

### DISCUSSION THREAD

**Discussion by antiperfectclub** [community/community#136525] [GitHub Education]
### Hi everyone,

Hello all,  hope you are well. I would like to follow up on the status of my student benefits progress. I lodged my request and it's been approved. 

Here is the proof. As it must be the busiest moment of the enrollment semester terms, I understand that this process may take longer. Just in case, I like to leave a note to raise a reminder on this :) 

<img width="714" alt="Screen Shot 2024-08-22 at 2 31 25 pm" src="https://github.com/user-attachments/assets/5f1b38f6-cb46-4f7c-967d-5f83be8129ec">

Thanks for your help in advance. Cheers.

**Comment by davevad93** [comment]
Hi @KimbaRiRi ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @KimbaRiRi was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by antiperfectclub** [reply]
It seems like I am not only one who's waiting. It's all good. Thanks for your reply. Cheers :)

**Reply by davevad93** [reply]
You're welcome @KimbaRiRi ! 👍
